code/modules/atmospherics/machinery/other/zvent.dm

```diff
@@ -1,3 +1,15 @@
+
+/*
+	Remie note:
+	I added proper multiZ, but this thing is ancient
+	I "updated" it, and it DOES work, but it's
+	not in RPDs, pipe dispensers etc and some of its
+	content (on/off, volume_rate) aren't implemented.
+
+	Use this for ruins and stuff I guess, or fix it up proper.
+*/
```

Comment:
By that I mean that cross-z connections should only happen between nodes. Making that work shouldn't be too difficult, but having a vent like this is unnecessary complexity for a redundant gain (pipe up connected to a vent is basically the same)